gatsby-node.js

```diff
@@ -10,18 +10,24 @@ const deepEqual = require('deep-equal');
  */
 function fetchAlgoliaObjects(index, attributesToRetrieve = ['modified']) {
   return new Promise((resolve, reject) => {
```

Comment:
you can directly return index.browseObjects now instead of recreating a promise (just keep the hits object before the return and add `.then(() => hits)` instead of resolving to hits

Reply:
Indeed 🎯 !